test/test.mk

```diff
@@ -1,29 +1,29 @@
 # this is a makefile used to show testcases of rabit
 .PHONY: all
 
```

Comment:
are we ok to re-enable pylocal_recover_10_10k now?

Reply:
I am considering refactor rabit python wrapper as some point in near future and consolidate some duplicated code in rabit and xgboost code base.

The observation is we got similar code in xgboost/python package as well as rabit itself. If we want to verify rabit python wrapper can work as we expected, we can utilize xgboost python tests.

Reply:
done